TaskHouseApi/Repositories/EmployerRepository.cs

```diff
@@ -7,110 +7,115 @@
 using TaskHouseApi.DatabaseContext;
 using TaskHouseApi.Model;
 
+
 namespace TaskHouseApi.Repositories
 {
-    public class UserRepository : IUserRepository
+    public class EmployerRepository : IEmployerRepository
+
     {
-        //cache the users in a thread-safe dictionary to improve performance
-        private static ConcurrentDictionary<int, User> userCache;
+
+        public EmployerRepository()
+        {
+        }
+
+
+        //cache the Employers in a thread-safe dictionary to improve performance
+        private static ConcurrentDictionary<int, Employer> EmployerCache;
 
         //reference to database context
         private PostgresContext db;
 
-        public UserRepository(PostgresContext db)
+        public EmployerRepository(PostgresContext db)
         {
             this.db = db;
 
-            //populates userCache
-            if (userCache == null)
+            //populates EmployerCache
+            if (EmployerCache == null)
             {
-                userCache = new ConcurrentDictionary<int, User>(
-                    db.Users.ToDictionary(c => c.Id));
+                EmployerCache = new ConcurrentDictionary<int, Employer>(
+                    db.Employers.ToDictionary(c => c.Id));
             }
         }
 
-        public async Task<User> CreateAsync(User u)
+
+        public async Task<Employer> Create(Employer e)
         {
-            //add user to database using EF core
-            EntityEntry<User> added = await db.Users.AddAsync(u);
+            //add Employer to database using EF core
+            EntityEntry<Employer> added = await db.Employers.AddAsync(e);
 
             int affected = await db.SaveChangesAsync();
```

Comment:
Could we rename this method?
```suggestion
            int affected = await db.SaveChanges();
```